### PR TITLE
Integration test for revision history limit

### PR DESCRIPTION
**Pull request by hrishin**
Added integration test for #1017

### CODE REVIEW

File: it/src/it/revisionhistory/invoker.properties [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+# implied.  See the License for the specific language governing
+# permissions and limitations under the License.
+#
+
+invoker.goals.1=clean fabric8:resource
+invoker.mavenOpts=-Dfabric8.verbose -Dfabric8.mode=kubernetes -Dfabric8.install.batch.mode=true
```
Comment (nicolaferraro) [inline it/src/it/revisionhistory/invoker.properties]: Is this needed (-Dfabric8.install.batch.mode)?

Comment (hrishin) [reply it/src/it/revisionhistory/invoker.properties]: :D indeed not

**Comment by nicolaferraro** [comment]
[merge]

**Comment by nicolaferraro** [comment]
Thanks a lot!

**Pull request merged**